modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1193 @@
+
+
+==== Support annotating certain block elements directly when using the editor annotator API
+
+Several new block elements can now have comments attached to them, including:
+
+* Tables of contents
+* Images
+* Images with captions
+* Code samples
+* Media in iframes
+* Embedded media
+* Embedded pages
+
+When comments are showing, a 3px wide colored rectangle presents around any element with a comment attached. By default this rectangle presents in two shades of yellow: one for commented elements which are selected and one for commented elements which are not selected. Using a skin can, however, change this default.
+
+
+==== The Comments sidebar now presents with a title bar
+
+When the Comments sidebar is showing, it now presents with a title bar to visually distinguish the sidebar from the {productname} document proper.
+
+The title bar label is *Comments* in English and the appropriately translated equivalent has been added to all (productname} supported languages.
+
+
+==== The Comment text-entry field placeholder text has been changed
+
+The new comment sidebar text-entry field contains placeholder text.
+
+Previously, this placeholder string was, in English, `Say something…`.
+
+For this update, the English-language string has been changed to `Add comment…`.
+
+Updated translations of this placeholder text have also been added to all (productname} supported languages.
+
+
+==== The label name for the Save Comment button has been changed and can now be translated
+
+Previously, the Save Comment was labeled *Save*. After field reports that end-users were interpreting this to mean the entire {productname} document would be saved on pressing this button, two changes were made.
+
+1. The default label for this button is now *Comment*.
++
+This new default presents for both new comments and replies to existing comments.
+
+2. This default label can now be translated.
+
+As is usual in {productname}, translation of this string is done wth `addI18n()`:
+
+[source,js]
+----
+tinymce.addI18n('<optional-IETF-language-code>', {
+   'Comment': '<translation>'
+});
+----
+
+NOTE: When a comment is being edited, the default button, which writes the changes back and closes the Edit Comment dialog, is still labeled *Save*.
+
+
+==== The `change` event was not fired when deleting all conversations
+
+Previously, when all comments were deleted, existing undo levels were re-written but no new undo level was created. As a consequence no `change` event was fired.
+
+This resulted in applications that relied on {productname} reporting its current state incorrectly assuming the now-deleted comments still existed.
+
+This update corrects this. In Comments 3.1.0, deleting any or all comments fires a `change` event. And a `changed` message is, consequently, printed to the console.
```

Comment:
```suggestion
This update corrects this. In Comments 3.1.0, deleting any or all comments fires a `change` event.
```
This doesn't normally happen, it'd only be shown if you had an event listener setup to log.